src/components/TodoFilter/TodoFilter.tsx

```diff
@@ -1,11 +1,49 @@
-export const TodoFilter = () => (
+import React from 'react';
+import { SortType } from '../../types/SortType';
+
+type Props = {
+  sortBy: SortType;
+  inputValue: string;
+  setInputValue: (value: string) => void;
+  setSortBy: (value: SortType) => void;
+};
+
+export const TodoFilter: React.FC<Props> = (
+  {
+    sortBy,
+    inputValue,
+    setSortBy,
+    setInputValue,
+  },
+) => (
   <form className="field has-addons">
     <p className="control">
       <span className="select">
-        <select data-cy="statusSelect">
-          <option value="all">All</option>
-          <option value="active">Active</option>
-          <option value="completed">Completed</option>
+        <select
+          data-cy="statusSelect"
+          value={sortBy}
+          onChange={(event: React.ChangeEvent<HTMLSelectElement>) => {
+            setSortBy(event.target.value as SortType);
+          }}
+        >
+          <option
+            value="all"
+
+          >
+            All
+          </option>
```

Comment:
```suggestion
          <option value="all">
            All
          </option>
```
Also, would be great to use Enum instead of the hardcoded values